p4-16/spec/P4-16-spec.mdk

```diff
@@ -2596,7 +2596,7 @@ types.
 
 #### Set types { #sec-set-types }
 
-The type `set<T>` describes _sets_ of values of type `T`. Set
+The type `set<T>` describes _sets_ of values of some numeric type `T` (section [#sec-numeric-values]). Set
```

Comment:
You can have sets of any other types.

Reply:
@MollyDream can you address this? We're blocked on merging this until it's fixed.